app/support/order_row_importer.rb

```diff
@@ -160,6 +166,11 @@ def validate_account
     end
   end
 
+  def validate_headers
+    unkown_headers = (@row.headers - HEADERS.values)
+    add_error("Unknown headers: #{unkown_headers.join('/')}") if unkown_headers.present?
```

Comment:
Spelling of `unknown`. Also,  `/` is probably not a good separator because "Netid / Email" is one of the headers.